chatsky/conditions/llm.py

```diff
@@ -0,0 +1,20 @@
+from chatsky.llm.methods import BaseMethod
+from chatsky.core.script_function import BaseCondition
+
+
+class LLMCondition(BaseCondition):
+
+    def __call__(model_name: str, prompt: str, method: BaseMethod):
```

Comment:
Should be `call(self, ctx) -> bool`.
`model_name, prompt, method` should be model fields.